### DISCUSSION TITLE
Simpson & Partners OCPP support

### DISCUSSION THREAD

**Discussion by bleara** [lbbrhzn/ocpp#949] [Q&A]
I have been in discussion with Simpson & Partners and they tell me they are working on an imminent release of the Home 7kW Chargers software with OCPP 1.6 incorporated. Is it just a documentaion change reflecting HA support for this device or is there code that needs to change too?

**Comment by lbbrhzn** [comment]
When a charger is OCPP 1.6 compliant then the HA integration should support it without code changes. The features that are supported through OCPP depend on the OCPP feature profiles that are supported by the charger. You can select the feature profiles when you add a new device to home assistant.

Unfortunately, some vendors claim their products are OCPP compatible, when they are in fact non-compliant. The integration allows bypass protocol validation in order to interact with those chargers, but you will get error notifications and limited functionality. We recommend forking the project when code changes are needed for a non-compliant charger.
